src/NServiceBus.AcceptanceTests/Basic/When_handling_inheriting_messages.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace NServiceBus.AcceptanceTests.Basic
+{
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_handling_inheriting_messages : NServiceBusAcceptanceTest
```

Comment:
Nitpick: This doesn't read well, how about "When_messagehandlers_share_the_same_class", "Should_not_share_state" or something?